docs/release-notes/NuGet-7.0.md

```diff
@@ -14,9 +14,18 @@ NuGet distribution vehicles:
 | NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
 | :---|:---|:---|
 | [**7.0.0**](https://nuget.org/downloads) | [Visual Studio 2026 version 18.0.0](https://visualstudio.microsoft.com/downloads/) | [10.0.100](https://dotnet.microsoft.com/download/dotnet/10.0)<sup>1</sup> |
+| [**7.0.1**](https://nuget.org/downloads) | [Visual Studio 2026 version 18.1.0](https://visualstudio.microsoft.com/downloads/) | [10.0.100](https://dotnet.microsoft.com/download/dotnet/10.0)<sup>1</sup> |
```

Comment:
The 10.0.100 SDK ships NuGet 7.0.0, not 7.0.1. If we haven't inserted NuGet into the SDK, then I'd leave that cell of the table empty.